vnext/ReactWindowsCore/DevSettings.h

```diff
@@ -102,6 +102,10 @@ struct DevSettings {
   // Until the ABI story is addressed we'll use this instead of the above for
   // the purposes of selecting a JSI Runtime to use.
   JSIEngineOverride jsiEngineOverride{JSIEngineOverride::Default};
+
+  /// A directory which the executor may read or write to in order to perform
+  /// its needed functions (e.g. bytecode caching)
+  std::filesystem::path executorScratchDirectory;
```

Comment:
>executorScratchDirectory [](start = 24, length = 24)

This does make me think a bit more. We can dynamically change Executors, so this directory would be reused unless the person setting this accounted for the difference (which they shouldn't). Not sure this matters much for now.

For that matter, the creator of an instance shouldn't even know what an executor is. The thing setting this specializes based on some executor specific things already though, so I think this is okay as is.